### DISCUSSION TITLE
Highlight focussed window (à la Tinkle)

### DISCUSSION THREAD

**Discussion by NeighNeighNeigh** [rxhanson/Charmstone-Community#29] [Ideas]
Just floating an idea here… 
My guess is this would only be possible if you moved out of sandbox… but it would be nice to have some kind of a visual effect to draw the eye to the newly focussed window, something similar to [Tinkle](https://github.com/pqrs-org/Tinkle).

**Comment by rxhanson** [comment]
Thanks for the idea. Indeed, this would require being outside of the sandbox. I have plans to implement a Charmstone-esque window switcher in rectangle pro where this would fit nicely. There are a few other items in the backlog before that, though.
